package.json

```diff
@@ -8,9 +8,10 @@
     "lint": "eslint src client tools",
     "lintfix": "eslint --fix src client tools",
     "checkdeps": "node tools/dependencyValidator.js",
-    "pr": "npm run typecheck && npm run lint && npm run test && npm run checkdeps",
+    "checklangs": "node tools/translationValidator.js",
+    "pr": "npm run typecheck && npm run lint && npm run test && npm run checkdeps && npm run checklangs",
     "watch": "tsc-watch --pretty --onSuccess \"npm run watchonsuccess\"",
-    "watchonsuccess": "npm run lintfix && npm run checkdeps"
+    "watchonsuccess": "npm run lintfix && npm run checkdeps && npm run checklangs"
```

Comment:
would prefer to run lintfix when I need it (aka on demand) and have "lint only" being used by default -- because lintfix modifies "my files" on every save and thus gives visual studio code the hick-ups sometimes.

Reply:
yep, however this is not what this pr is about ;)